drivers/i2c/i2c_mchp_xec_v2.c

```diff
@@ -77,11 +79,13 @@ struct xec_speed_cfg {
 	uint32_t start_hold_time;
 	uint32_t config;
 	uint32_t timeout_scale;
+	uint32_t idle_scale;
```

Comment:
Where is this new parameter used?

Reply:
This parameter is used to program the idle scaling register. There was already the `config `field used for that. I will remove that field and use `idle_scale `instead to make it more clear.

Reply:
updated.